targets/ESP32/_nanoCLR/nanoFramework.Espnow/nanoFramework_espnow_native_nanoFramework_Espnow_EspNowController.cpp

```
@@ -0,0 +1,206 @@
+
+HRESULT Library_nanoFramework_EspNow_native_nanoFramework_EspNow_EspNowController::
+    NativeEspNowSend___I4__SZARRAY_U1__SZARRAY_U1__I4(CLR_RT_StackFrame &stack)
+{
+    NANOCLR_HEADER();
+
+    DEBUG_FENTER();
+
+    esp_err_t ret;
+
+    CLR_RT_HeapBlock_Array *peerMacArg = stack.Arg1().DereferenceArray();
+    char *peerMac = (char *)peerMacArg->GetFirstElement();
```

Clang (14.0.6)

[warning] do not use C-style cast to convert between unrelated types

(cppcoreguidelines-pro-type-cstyle-cast)